webapp/src/Service/ImportProblemService.php

```diff
@@ -190,6 +191,62 @@ public function importZippedProblem(
             }
         }
 
+        if ($deleteOldData && $problem->getProbid()) {
+            if (abs($problem->getTimelimit() - $defaultTimelimit) > 0.001) {
+                $problem->setTimelimit($defaultTimelimit);
+                $messages[] = 'Resetting time limit.';
+            }
+            if ($problem->getCompareExecutable()) {
+                $problem->setCompareExecutable();
+                $messages[] = 'Clearing compare executable.';
+            }
+            if ($problem->getSpecialCompareArgs()) {
+                $problem->setSpecialCompareArgs('');
+                $messages[] = 'Clearing compare arguments.';
+            }
+            if ($problem->getRunExecutable()) {
+                $problem->setRunExecutable();
+                $messages[] = 'Clearing run executable.';
+            }
+            if ($problem->getCombinedRunCompare()) {
+                $problem->setCombinedRunCompare(false);
+                $messages[] = 'Clearing combined run and compare script flag.';
+            }
+            if ($problem->getMemlimit()) {
+                $problem->setMemlimit(null);
+                $messages[] = 'Clearing memory limit.';
+            }
+            if ($problem->getOutputlimit()) {
+                $problem->setOutputlimit(null);
+                $messages[] = 'Clearing output limit.';
+            }
+            if ($problem->getProblemtext()) {
```

Comment:
.... or problemtexttype is null?

Reply:
Yep, not null. I mean it should never happen, but do you want to rely here on the contract between the two?

Reply:
Doesn't hurt to add it I guess